modules/ROOT/pages/con_pipeline-git-resources.adoc

```diff
@@ -0,0 +1,84 @@
+[]
+= Git Resource
+
+Git resource represents a [git](https://git-scm.com/) repository, that contains
+the source code to be built by the pipeline. Adding the git resource as an input
+to a Task will clone this repository and allow the Task to perform the required
+actions on the contents of the repo.
```

Comment:
We can make this statement more direct and user-focused. For example:
When you add the git resource as an input to a Task, the Task clones the repository and performs the required
actions on the contents of the repository.